### PR TITLE
gradle: update dependencies

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**
remove retrolambda as it is no longer needed

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -32,8 +31,8 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     implementation "com.android.support:appcompat-v7:$libraries.support"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
-    implementation "io.reactivex:rxandroid:$libraries.rxAndroid"
```
Comment (dkhmelenko) [inline app/build.gradle:36]: We need to keep RxAndroid dependency.

File: miband-sdk/build.gradle [hunk 2]
```diff
@@ -29,7 +28,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     implementation "com.android.support:appcompat-v7:$libraries.support"
-    implementation "io.reactivex:rxandroid:$libraries.rxAndroid"
```
Comment (dkhmelenko) [inline miband-sdk/build.gradle:32]: We need to keep RxAndroid

File: miband-sdk/build.gradle [hunk 3]
```diff
@@ -29,7 +28,7 @@ dependencies {
...
+    implementation "io.reactivex.rxjava2:rxandroid:$libraries.rxAndroid"
```
Comment (dkhmelenko) [inline miband-sdk/build.gradle:31]: This entry duplicates previous one

**Review by dkhmelenko — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (SubhrajyotiSen) [reply app/build.gradle:36]: I haven't removed RxAndroid. According to their docs, the complete dependency name is `io.reactivex.rxjava2:rxandroid` and not `io.reactivex:rxandroid`. I just updated that

**Review by dkhmelenko — APPROVED** [review]

**Pull request merged**